src/_tests/fixtures/43144/mutations.json

```diff
@@ -33,7 +33,7 @@
     "variables": {
       "input": {
         "subjectId": "MDExOlB1bGxSZXF1ZXN0Mzg4NDkxOTI2",
-        "body": "@jeffreymeng Everything looks good here. Great job! I am ready to merge this PR (at f1f5c4b) on your behalf.\n\nIf you'd like that to happen, please post a comment saying:\n\n> Ready to merge\n\nand I'll merge this PR almost instantly. Thanks for helping out! :heart:\n\n(@pocesar: you can do this too.)\n<!--typescript_bot_merge-offer-->"
+        "body": "@jeffreymeng Everything looks good here. Great job! I am ready to merge this PR (at f1f5c4bb0ae553f56766882f6458d2e22baa87c7) on your behalf.\n\nIf you'd like that to happen, please post a comment saying:\n\n> Ready to merge\n\nand I'll merge this PR almost instantly. Thanks for helping out! :heart:\n\n(@pocesar: you can do this too.)\n<!--typescript_bot_merge-offer-->"
```

Comment:
GitHub abbreviates these anyway

Reply:
I'd like to keep this abbreviated still.  One reason is that this verbosity is much less convenient (eg, when debugging); another reason is that it changes the text that is being searched, so if this is merged when such comments exist, the new searches won't work.

Reply:
Done :heavy_check_mark: